templates/base/packages/nextjs/package.json

```diff
@@ -20,7 +20,7 @@
     "@uniswap/sdk-core": "~4.0.1",
     "@uniswap/v2-sdk": "~3.0.1",
     "blo": "~1.0.1",
-    "burner-connector": "~0.0.3",
+    "burner-connector": "^0.0.3",
```

Comment:
also btw, 0.0.4 is latest version which we launched yesterday(but doesn't have any changes to logic it was just package.json git url changes). Keeping this as `^0.0.3`  since in SE-2 we are using that 

using `~0.0.4` also gives module not found error

Reply:
Damn, weird. Sorry I didn't test it before (didn't expect that to break haha).

This PR fixes it.

Reply:
>  (didn't expect that to break haha).

Same :( !! lol